tests/cupyx_tests/scipy_tests/special_tests/test_exp1.py

```diff
@@ -0,0 +1,41 @@
+import cupyx.scipy.special  # NOQA
+
+from cupy import testing
+from cupy.testing import numpy_cupy_allclose
+
+
+@testing.gpu
+@testing.with_requires("scipy")
+class TestExp1:
+
```

Comment:
Hi, just a minor note. I think `@testing.gpu` decorator can be removed as it was conventionally used. Right, @emcastillo!?

Reply:
With this attrubute pytest deselects these tests if we want to run tests without GPU like this `py -m pytest tests/cupyx_tests/scipy_tests/special_tests/test_expi.py -m='not gpu'` . After I removed `@testing.gpu` the same command run these tests. As for me it seems like correct behavour, but I may be wrong

Reply:
Yeah, we actually disabled the `not gpu` marker since merely import cupy requires a GPU, so it is deprecated now :).

Reply:
So, should I remove @testing.gpu and add nothing new to imports?

Reply:
thats it!

Reply:
done!